jina/drivers/__init__.py

```diff
@@ -238,6 +245,15 @@ def __init__(self, traversal_paths: Tuple[str] = ('c', 'r'), *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._traversal_paths = [path.lower() for path in traversal_paths]
 
+    def _apply_root(
+        self,
+        docs: 'DocumentSet',
+        field: str,
```

Comment:
if we say is `None`, should we also hide `field`?

Reply:
`field` contains `"docs"`.
The `context_doc` is set to `None`